pinax/messages/forms.py

```diff
@@ -21,7 +21,7 @@ def label_from_instance(self, obj):
 class NewMessageForm(forms.ModelForm):
 
     subject = forms.CharField()
-    to_user = UserModelChoiceField(queryset=get_user_model().objects.none)
+    to_user = UserModelChoiceField(queryset=get_user_model().objects.none())
```

Comment:
This is going to be an object that lives globally for the life of the process.  Probably won't matter because the queryset is getting set for each new instance in the `__init__` however, it does feel dirty to me.

Reply:
It was producing an error in my build that was preventing the message_create view from being displayed.  I narrowed it down to the form creation and specifically the queryset.  I'm not sure how else you'd propose to fix this but I'm all ears.